.github/workflows/build_and_test.yml

```diff
@@ -73,7 +81,7 @@ jobs:
       - run: docker images --all
 
       - name: Test Enclave # cargo test is not supported in the enclave, see: https://github.com/apache/incubator-teaclave-sgx-sdk/issues/232
-        run: docker run --name ${{ env.BUILD_CONTAINER_NAME }} integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }} test --all
+        run: docker run --device=/dev/sgx/enclave --device=/dev/sgx/provision -v /var/run/aesmd:/var/run/aesmd integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }} test --all
```

Comment:
This also needs to be configurable, such that it is only included if `${{ matrix.sgx_mode == "HW" }}`